src/renderer/components/FtKeyboardShortcutPrompt/FtKeyboardShortcutPrompt.vue

```diff
@@ -0,0 +1,184 @@
+        : ['NAVIGATE_TO_HISTORY', t('KeyboardShortcutPrompt.Navigate to History')]
+    ),
+    ['NEW_WINDOW', t('KeyboardShortcutPrompt.New Window')],
+    ['MINIMIZE_WINDOW', t('KeyboardShortcutPrompt.Minimize Window')],
+    ['CLOSE_WINDOW', t('KeyboardShortcutPrompt.Close Window')],
+    ['RESTART_WINDOW', t('KeyboardShortcutPrompt.Restart Window')],
+    ['FORCE_RESTART_WINDOW', t('KeyboardShortcutPrompt.Force Restart Window')],
+    ['TOGGLE_DEVTOOLS', t('KeyboardShortcutPrompt.Toggle Developer Tools')],
+    ['RESET_ZOOM', t('KeyboardShortcutPrompt.Reset Zoom')],
+    ['ZOOM_IN', t('KeyboardShortcutPrompt.Zoom In')],
+    ['ZOOM_OUT', t('KeyboardShortcutPrompt.Zoom Out')],
+    ['FOCUS_SEARCH', t('KeyboardShortcutPrompt.Focus Search')],
+    ['SEARCH_IN_NEW_WINDOW', t('KeyboardShortcutPrompt.Search in New Window')],
+
+    ['REFRESH', t('KeyboardShortcutPrompt.Refresh')],
+    ['FOCUS_SECONDARY_SEARCH', t('KeyboardShortcutPrompt.Focus Secondary Search')],
+
+    ['CAPTIONS', t('KeyboardShortcutPrompt.Captions')],
+    ['THEATRE_MODE', t('KeyboardShortcutPrompt.Theatre Mode')],
+    ['FULLSCREEN', t('KeyboardShortcutPrompt.Fullscreen')],
+    ['FULLWINDOW', t('KeyboardShortcutPrompt.Full Window')],
+    ['PICTURE_IN_PICTURE', t('KeyboardShortcutPrompt.Picture in Picture')],
+    ['MUTE', t('KeyboardShortcutPrompt.Mute')],
+    ['VOLUME_UP', t('KeyboardShortcutPrompt.Volume Up')],
+    ['VOLUME_DOWN', t('KeyboardShortcutPrompt.Volume Down')],
+    ['TAKE_SCREENSHOT', t('KeyboardShortcutPrompt.Take Screenshot')],
+    ['STATS', t('KeyboardShortcutPrompt.Stats')],
+
+    ['PLAY', t('KeyboardShortcutPrompt.Play')],
+    ['LARGE_REWIND', t('KeyboardShortcutPrompt.Large Rewind')],
+    ['LARGE_FAST_FORWARD', t('KeyboardShortcutPrompt.Large Fast Forward')],
+    ['SMALL_REWIND', t('KeyboardShortcutPrompt.Small Rewind')],
+    ['SMALL_FAST_FORWARD', t('KeyboardShortcutPrompt.Small Fast Forward')],
+    ['DECREASE_VIDEO_SPEED', t('KeyboardShortcutPrompt.Decrease Video Speed')],
+    ['INCREASE_VIDEO_SPEED', t('KeyboardShortcutPrompt.Increase Video Speed')],
+    ['SKIP_N_TENTHS', t('KeyboardShortcutPrompt.Skip by Tenths')],
+    ['LAST_CHAPTER', t('KeyboardShortcutPrompt.Last Chapter')],
+    ['NEXT_CHAPTER', t('KeyboardShortcutPrompt.Next Chapter')],
+    ['LAST_FRAME', t('KeyboardShortcutPrompt.Last Frame')],
+    ['NEXT_FRAME', t('KeyboardShortcutPrompt.Next Frame')],
+  ])
+})
+
+function hideKeyboardShortcutPrompt() {
+  store.dispatch('hideKeyboardShortcutPrompt')
+}
+
+function getLocalizedShortcutNamesAndValues(dictionary) {
+  return Object.entries(dictionary)
+    .filter(([key]) =>
+      localizedShortcutNameDictionary.value.has(key)
+    )
+    .map(([shortcutNameKey, shortcut]) => {
+      const localizedShortcutName = localizedShortcutNameDictionary.value.get(shortcutNameKey)
+      const localizedShortcut = getLocalizedShortcut(shortcut)
+      return [localizedShortcutName, localizedShortcut]
+    })
+}
```

Comment:
As you are accessing the value quite a few times, it makes sense to cache it at the top of the function.

```suggestion
function getLocalizedShortcutNamesAndValues(dictionary) {
  const nameDictionary = localizedShortcutNameDictionary.value

  return Object.entries(dictionary)
    .filter(([key]) =>
      nameDictionary.has(key)
    )
    .map(([shortcutNameKey, shortcut]) => {
      const localizedShortcutName = nameDictionary.get(shortcutNameKey)
      const localizedShortcut = getLocalizedShortcut(shortcut)
      return [localizedShortcutName, localizedShortcut]
    })
}
```